hack/debug.sh

```diff
@@ -468,6 +502,8 @@ for systemd_dir in /etc/systemd/system /usr/lib/systemd/system; do
     for name in openshift origin atomic-openshift; do
 	if [ -f $systemd_dir/$name-master.service ]; then
 	    aos_master_service=$name-master.service
+	    aos_master_controllers_service=$name-master-controllers.service
+	    aos_master_api_service=$name-master-api.service
```

Comment:
Are these service names going to change due to new branding? (openshift local/lab/enterprise)